website/docs/d/workspace_outputs.html.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_workspace_outputs"
+sidebar_current: "docs-datasource-tfe-state-outputs"
+description: |-
+  Retrieves the State outputs per organization and workspace.
+---
+# Data Source: tfe_workspace_outputs
+
+This data source is used to retrieve the state outputs for a given workspace.
+It enables values in the outputs to be used dynamically in a terraform
+configuration.
+
+## Example Usage
+
+Using the `tfe_workspace_outputs` data source in a terraform configuration.
+
+In the example below, assume we have a state outputs that looks like this:
+
+```
+{
+  "version": <version>,
+  "terraform_version": "<terraform-version>",
+  ...
+  "outputs": {
+    "identifier": {
+      "value": "9023256633839603543",
+      "type": "string"
+    },
+    "records": {
+      "value": ["hashicorp.com", "terraform.io"],
+      "type": ["list", "string"]
+    },
+    "secret": {
+      "value": "token",
+      "type": "string",
+      "sensitive": true
+    }
+  },
+  "resources": [
+    ...
+  ]
+}
+```
```

Comment:
Generally speaking, the state file is an internal implementation detail who's format isn't guaranteed and has defined interfaces like `terraform state show` and outputs themselves to interact with it - in other words, I wouldn't actually give the content here as an example. It's prone to being outdated, and isn't the thing that people should care about when crafting their configurations. 

Instead, I would show the user-level interface used - configurations!

Example (yes, you can have a Terraform configuration with just hardcoded outputs 😆):

---

Given the outputs defined in a configuration used at `my-org/my-workspace`:

```
output "foo" {
  value = "a"
}

output "bar" {
  value = "b"
}
```

These outputs can be used in another:

```
data "tfe_workspace_outputs" "external_outputs" {
  organization = "my-org"
  workspace = "my-workspace"
}

<rest of example omitted but you get my gist>
```

Reply:
Right, will update